src/clj/puppetlabs/services/puppet_admin/puppet_admin_service.clj

```diff
@@ -42,5 +44,5 @@
         (core/build-ring-handler route
                                  whitelist-settings
                                  jruby-service
-                                 wrap-with-authorization-check)))
+                                 (fn [request] (wrap-with-authorization-check request {:oid-map oid-map})))))
```

Comment:
It's worth noting here, I think, that we have a "master service" in PE as well, so any changes made in this namespace will need to applied there too.